recipes/geograpy3/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "geograpy3" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geograpy3-{{ version }}.tar.gz
+  sha256: 52e7a65a4f39b3fbe84f752814b701bb4093798b22c638622fbcd3445d3a862e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Since this is pure Python, it would be good to make this noarch:
```suggestion
  number: 0
  noarch: python
```

---

Added!